test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/io/CharArrayReaderTest.scala

```diff
@@ -0,0 +1,166 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.io
+
+import java.io._
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.AssertThrows._
+
+class CharArrayReaderTest {
+  private val hw: Array[Char] = Array('H', 'e', 'l', 'l', 'o', 'W', 'o', 'r', 'l', 'd')
+
+  private def withClose[T <: AutoCloseable](closeable: T)(fn: T => Unit): Unit = {
+    fn(closeable)
+
+    if (closeable != null)
+      closeable.close()
+  }
+
+  @Test def should_support_constructor_char_array(): Unit =
```

Comment:
Since the body of this method is now multi-line, the method should have `{}`, according to [our coding style](https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#braces):
```suggestion
  @Test def should_support_constructor_char_array(): Unit = {
```
The same applies to a bunch of other test methods.